README.md

```diff
@@ -6,41 +6,42 @@ The key words MUST, MUST NOT, SHOULD, SHOULD NOT, and MAY/OPTIONAL, in these doc
 
 ## Contents
 
-### Culture
-
+### Core Values
 * [Engineering Values](culture/values.md)
 * [Library Values](culture/library-values.md)
 
-### Standards
-
-* [Lambda deployment with node-lambda](standards/node-lambda.md)
+### Policies
+* [Accessibility](standards/accessibility.md)
+* [Privacy & Data Retention](standards/privacy.md)
+  
+### Code Standards
 * [Coding Style](standards/coding-standards.md)
-* [Logging](standards/logging.md)
-* [Monitoring & Alarms](standards/alerting.md)
-* [Naming Conventions](standards/naming-conventions.md)
-* [Test Coverage](standards/test-coverage.md)
-* [CI Coverage](standards/ci-and-deployment.md)
-* [Travis CI](standards/travis-ci.md)
+    * [Versioning](standards/versioning.md)
+    * [Naming Conventions](standards/naming-conventions.md)
+    * [Logging](standards/logging.md)
+    * [Test Coverage](standards/test-coverage.md)
+    * [Documentation](standards/documentation.md)
+  * [Javascript]()
+    * [NYPL React Design System]()
+  * [Python]()
+  * [Ruby on Rails]() 
+  * [PHP]()
 * [Peer Review](standards/peer-review.md)
-* [Accessibility](standards/accessibility.md)
-* [Documentation](standards/documentation.md)
-* [Versioning](standards/versioning.md)
+* [CI Coverage](standards/ci.md)
+  * [Github Actions]()
+  * [Travis CI](standards/travis-ci.md) 
+* [Deployment](standards/deployment.md)
+  * [AWS](standards/aws.md)
+  * [Node Lambda](standards/node-lambda.md)
 * [Production Readiness](standards/production-readiness.md)
+* [Monitoring & Alarms](standards/alerting.md)
 * [Post-mortems](standards/postmortems.md)
```

Comment:
I'm so-so on the use of "post-mortem" as a term for this type of work. It's very accusatory in the same way `git blame` is (you've killed something and we want to know how/why). Smartsheet has some good alternates: https://www.smartsheet.com/content/business-post-mortem I like "critical incident report" because while it's less catchy it can be shortened (CIR) and is accurate without being too harsh.

Reply:
It seems like we're going to be replacing these with the root-cause-analysis documents, right?  I think I'll change it to that